app/views/tag/_tagging.html.erb

```diff
@@ -50,4 +50,4 @@ $(function () {
 </form>
 <% end %>
 
-<%= render partial: 'tag/location' %>
+<%= render partial: 'tag/location', locals:{parent: parent} %>
```

Comment:
This would also not be needed if we could check the `params[:action]` variable above. Make sense?